README.md

```diff
@@ -150,7 +151,7 @@ Now the files are ready to be translated in Transifex.
 
 4. **To fetch new translations** when they're done, you need to run `tx pull -a` to fetch all, or `tx pull -l ru` to fetch a particular language.
 
-5. **Commit** the new or updated .po files in `docs/locale`, using a separate commit from your edits to the source (RST, JSON or CSV) files.
+5. If you are still on the main branch, check out a new development branch from which you will make a PR with the updated translations. **Commit** the new or updated .po files in `docs/locale`.
```

Comment:
Is this better than checking out a development branch before fetching the new translations?

Reply:
It doesn't make any difference which way around you do it. I expect you're most likely to *already* be on a development branch, and this is just in case you aren't.